NEWS.md

```diff
@@ -1,6 +1,32 @@
 MathOptInterface (MOI) release notes
 ====================================
 
+v0.9.15 (September 10, 2020)
+---------------------------
+
+For a detailed list of the closed issues and pull requests from this release,
+see the [tag notes](https://github.com/jump-dev/MathOptInterface.jl/releases/tag/v0.9.15).
+
+A summary of changes are as follows:
+
+- Bridges improvements:
+  - (R)SOCtoNonConvexQuad bridge
+  - ZeroOne bridge
+  - Add `supports_add_constrained_variable`
```

Comment:
It was already there. The difference is that it's now used by `LazyBridgeOptimizer`